osh/word_parse.py

```diff
@@ -269,6 +269,8 @@ def _ParseVarExpr(self, arg_lex_mode):
     if self.token_type == Id.Right_VarSub:
       return part  # no ops
 
+    log('%s', self.token_type)
```

Comment:
remove?

Reply:
Hm not sure when that snuck in, I pushed a change to remove it.